pages/api/startPlayer/index.js

```diff
@@ -0,0 +1,47 @@
+import { getToken } from "next-auth/jwt";
+
+export default async function handler(req, res) {
```

Comment:
please use more descriptive names for your parameters, such as `request` and `response`

this will cause a naming conflict with line 26 though, but you will figure something out ;)